### ISSUE TITLE
[Bug]: Cannot run Python wordcount quickstart example, ModuleNotFoundError

### ISSUE THREAD

**Issue by demongolem-biz** [apache/beam#23758]
### What happened?

WIth python 3.8 on rhel 8, I ran `python3.8 -m apache_beam.examples.wordcount`.  I did this from a fresh virtual environment where I pip installed apache-beam.

The error I get is `ModuleNotFoundError: No module named 'apache_beam.portability.api`.  This exception is raised in `apache_beam/portability/common_urns.py` on line 22 with the line `from .api import beam_runner_api_pb2_urns`

I was following the instructions on [Apache Beam Get Started](https://beam.apache.org/get-started/quickstart-py) to attempt to run the example code.

### Issue Priority

Priority: 2

### Issue Component

Component: examples-python

**Comment by tvalentyn** [comment]
Are you by chance a Macbook M1 user ?

**Comment by tvalentyn** [comment]
nvm, i saw you mentioned rhel 8

**Comment by tvalentyn** [comment]
would you mind posting full sequence of commands you run?

**Comment by tvalentyn** [comment]
Is it reproducible with some docker container?

**Comment by tvalentyn** [comment]
cc: @BjornPrime who encountered a similar issue on an M1 machine.

**Comment by BjornPrime** [comment]
You might try the process described here: https://github.com/apache/beam/issues/22742#issuecomment-1218216468

It was intended for M1 Macs but since you seem to be having a similar issue, it might be worth a try. Let us know whether or not it works for you.

**Comment by demongolem-biz** [comment]
Sorry don't have access to the machine currently (later today will).  However I did click on that link and I noticed protobuf which has been an issue for us before.  We generally go with 3.17.3 and the link says 3.20.1 and so it could be that.  Will find out more and report back.

**Comment by demongolem-biz** [comment]
No, getting protobuf to a higher version does not change the error.

**Comment by BjornPrime** [comment]
Did you run `python gen_protos.py` once you upgraded protobuf?

**Comment by demongolem-biz** [comment]
No I hadn't and that did get me past that particular error with protobuf 3.20.3 installed.  Yes now I can successfully run on my own provided .txt file.  Thanks.

**Comment by tvalentyn** [comment]
@BjornPrime would you mind adding helpful info to s.apache.org/beam-python-dev-wiki ?

**Comment by tvalentyn** [comment]
(you can communicate the M1 workarounds there)

**Comment by damccorm** [comment]
This got added to the 2.44 milestone via close/reopen and is probably not a real release blocker

**Comment by BjornPrime** [comment]
Added a link to https://github.com/apache/beam/issues/22742#issuecomment-1218216468 to the wiki, which seems to resolve the issue in many cases and is the closest thing we have to a solution at this point. If that doesn't work for them, hopefully the discussion on the issue will help or they can contribute their own experience to the discussion.

**Comment by BjornPrime** [comment]
.close-issue
